src/tasks/deploy.mk

```
@@ -86,11 +86,18 @@ deplo%: ## deploy: deploy the app to heroku
 	  nht configure $(VAULT_NAME) $(HEROKU_APP_US) --overrides REGION=US \
 	)
 
+deploy-promote: ## deploy-promote: promote the staging app to production.
```

I would have expected the target to be called `deploy-production`?

```suggestion
deploy-production: ## deploy-production: deploy to production by promoting the staging app
```

---

`deploy-production` is taken by the Canary deploy tasks

---

_Interesting_. Then I have no opinion here.

---

@sjparkinson Are you ok to approve this PR in that case?

---

I'm happy with this, but I'd also like to see someone from @Financial-Times/etg review this too.